nautobot/core/urls.py

```diff
@@ -96,11 +99,12 @@
         path("", include("django_prometheus.urls")),
     ]
 
-if settings.SOCIAL_AUTH_ENABLED:
+# Only add this setting if social-auth is installed
+if reduce(operator.or_, map(lambda s: s.startswith("social_core.backends"), settings.EXTERNAL_AUTH_BACKENDS), False):
```

Comment:
This lovely piece of code golf has my eyes-crossing. I know it's efficient but can we make it somewhat more readable?

Reply:
It's no more code golf than half the magic that is this thing called the django framework, but I digress and will concede to the imperatives of the repository overlords.